docs/guides/first-device-level-experiment.mdx

```diff
@@ -3,62 +3,149 @@ sidebar_label: Device-level Experiments
+---
 
-Click on the Create button and enter the name and description for your experiment. Click the checkbox for **Use Stable ID** check box and click **Create**.
+## Overview of Steps to Set Up a Device-level Experiment
 
-![Screen Shot 2022-06-26 at 10 43 49 AM](https://user-images.githubusercontent.com/74584483/175827163-7669bd8d-ee7f-49af-943a-b1f71d95428e.png)
+To create and implement a device-level experiment, you'll follow three main steps:
 
-You can fill out the Experiment **Scorecard** with a hypothesis and primary/secondary metrics, just as you would with any other experiment.
+1. **Set up your experiment in the Statsig Console**: In this step, you will create and configure the experiment, including defining the parameters and deciding on the allocation.
+2. **Integrate the experiment with the client SDK**: After setting up the experiment, you'll configure the Statsig SDK in your client-side application to evaluate the experiment using a stable ID.
+3. **(Optional) Update user information post-login**: If your users log in after the experiment begins, you can update their information to run user ID-based experiments without affecting the ongoing device-level experiment.
 
-Under the **Groups** card, add a parameter that you wish to experiment on.  For now, let's just add a simple boolean "enabled":
+---
+With this overview, let’s dive into the specific steps:
 
-![Screen Shot 2022-06-26 at 10 49 05 AM](https://user-images.githubusercontent.com/74584483/175827265-53e18e66-14ce-4d8e-a409-201bd6191615.png)
 
-By default, the experiment will be allocated to 100% of your user base in the **Allocation** panel. Adjust this as necessary - you may wish to start with a smaller rollout percentage, and ramp up the allocation after proving out the new variant.
+## Step 1: Setting Up a Device-level Experiment in the Statsig Console
 
-![image](https://user-images.githubusercontent.com/1315028/138972419-b7c42f97-29ec-407e-851f-3301130a21c5.png)
+This section covers how to create and configure a new device-level experiment in the Statsig Console.
 
-Finally, don't forget to click on the **Save** button at the top right hand side of the page to complete your experiment setup, and the **Start** button when you are ready to roll this out to production users!
+1. **Log in to the Statsig Console**  
+   Head to the [Statsig Console](https://console.statsig.com/), and navigate to **Experiments+** from the left-hand side.
 
+2. **Create a New Experiment**  
+   - Click the **Create** button.
+   - Enter a name and description for your experiment.
+   - Check the box for **Use Stable ID** to specify that this is a device-level experiment.
+   - Click **Create**.
 
-### Step 2 - Use your experiment configuration in the client-side of your application
+   ![Experiment Creation Screen](https://user-images.githubusercontent.com/74584483/175827163-7669bd8d-ee7f-49af-943a-b1f71d95428e.png)
```

Comment:
a) do we really need a screenshot?

b) if so, please update it - you reused a screenshot from June 2022